http-server/src/main/java/io/airlift/http/server/HttpServerConfig.java

```diff
@@ -225,6 +226,12 @@ public HttpServerConfig setKeystorePassword(String keystorePassword)
         return this;
     }
 
+    @AssertTrue(message = "PEM keystore path/password must be provided when HTTPS is enabled")
```

Comment:
I think the keystore may be in different format

```suggestion
    @AssertTrue(message = "keystore path/password must be provided when HTTPS is enabled")
```